frontend/src/app/academics/academics-home/academics-home.component.html

```diff
@@ -27,3 +27,31 @@
     </button>
   </mat-card-actions>
 </mat-card>
+
+<div
+  *ngIf="applicationService.user_application$ | async as application; else noApplication">
+  <mat-card class="content" appearance="outlined">
+    <mat-card-content>
+      <mat-card-subtitle>Become a TA!</mat-card-subtitle>
```

Comment:
The subtitle is a little small, so it may be nice to make this a title rather than a subtitle.